doc/quickstart-integratedsearchdisk.md

```diff
@@ -302,7 +302,7 @@ public class IndexingAndRetrievalExample {
 			int docid = doc.getDocid();
 			double score = doc.getScore();
 			String docno = doc.getMetadata("docno")
-			System.out.println("   Rank "+i+": "+docid+" "+docno+" "+score);
```

Comment:
Thanks for your efforts.

Any reason why you removed remove the rank here?

Reply:
It is restored now.

Reply:
I have made it work, but adding the i (now called rank) variable and updating it in the iterator. Do you agree?

Reply:
Looks good to me, thanks!